mods/christmas_craft/settings.lua

```diff
@@ -1,5 +1,5 @@
 
-print (" ---- Overrider christmas_craft = true! ---- ")
+print(" ---- Overrider christmas_craft = true! ---- ")
```

Comment:
Maybe we could use actual `minetest.log` calls here? Just suggesting

Reply:
Sounds good. 👍